signhandler/src/main/java/se/swedenconnect/signservice/signature/impl/DefaultSignatureHandler.java

```diff
@@ -16,20 +16,23 @@
 package se.swedenconnect.signservice.signature.impl;
 
 import java.security.SignatureException;
+import java.time.Duration;
```

Comment:
Remove. Unused. The compiler warns.

Isn't it possible to setup IntelliJ to remove unused imports when saving?

https://stackoverflow.com/questions/12585253/how-to-remove-unused-imports-in-intellij-idea-on-commit